src/dislocker.rb

```diff
@@ -11,10 +11,12 @@ class Dislocker < Formula
   version "0.4.0"
 
   depends_on "polarssl"
-  depends_on "osxfuse"
+  depends_on :osxfuse
 
   def install
+    system "mkdir -p #{prefix}/{bin,lib,share/man/man1}"
     system "make -C src"
     system "make -C src install DESTDIR=#{prefix}/"
+    system "cp #{prefix}/lib/libdislocker.0.dylib #{prefix}/bin/"
```

Comment:
I realize, I just wanted it to compile and work. A proper fix will definitely be nicer, this just makes it work for the time being. If I had time, I'd look deeper into the underlying issue and find the correct directory but unfortunately I don't. Hopefully you can commit the changes (once you clean this code a little more) so that Yosemite users can compile it with ease. El Capitan is just around the corner, I bet it's time this was fixed up for Yosemite :)